src/components/Button/Button.module.css

```diff
@@ -121,6 +121,106 @@
   }
 }
 
+/**
+ * Secondary button
+ * 1) Medium prominence button used for less important actions
+ */
+.button--secondary {
+  border-color: var(--eds-theme-color-primary-border);
+  background-color: var(--eds-theme-color-primary-background);
+  color: var(--eds-theme-color-body-foreground-inverted);
+
+  &:hover {
+    background-color: var(--eds-theme-color-primary-background-hover);
+    color: var(--eds-theme-color-body-foreground-inverted);
+  }
+
+  &:focus {
+    background-color: var(--eds-theme-color-primary-background-hover);
+    color: var(--eds-theme-color-body-foreground-inverted);
+  }
+
+  &:focus-visible {
+    @mixin focus;
```

Comment:
what's the difference between this & the `focus` styles above? (should there be a difference)?

Reply:
I believe they're only showing the actual focus outline when you tab to a button (as opposed to clicking on it with a mouse) because people typically find those outlines ugly and don't like to see them unless it's necessary for the user. I guess they're okay with the text and background color changes, but they don't like the outline.

This is [Along's implementation](https://github.com/chanzuckerberg/along/blob/c2f1572541dd1c2e35b4e35fa68c5f6ec44fda96/client/src/ui/theme/index.tsx#L47):
```
  /**
   * Hide focus outlines from mouse interaction, but still use them for keyboard interaction.
   * This relies on the https://github.com/WICG/focus-visible polyfill.
   */
  .js-focus-visible :focus:not(.focus-visible) {
    outline: none;
  }
```
They have that styling at the top level, and then in the actual component styles they just target `:focus`.

I kinda wanna just use `:focus` for now and make a ticket to come back and consider using this because there could be serious accessibility issues if we throw it in and mess up the implementation. `traject` doesn't currently use this, so there's no regression if we skip it.

Reply:
Mmm but it's used in 21 places in 14 files. 🤔